larray/core.py

```diff
@@ -7667,26 +7667,67 @@ def opmethod(self, other):
     __ror__ = _binop('ror')
 
     def __matmul__(self, other):
+        """
+        What doc of Numpy matmul says:
```

Comment:
Well, my first intention was not to write a real documentation but just to add some indications to remember how __matmul__ from Numpy works. I'll move that part inside the function.